modules/userdirectory-canvas/src/main/resources/OSGI-INF/canvas-user-factory.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oof, it looks nice, but I think that it would have to be someone a bit more experienced to take a look at implementing this.  We no longer have an active Opencast/Canvas implementation to test against, so I don't want to make any breakable changes.